src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1754,66 +1755,89 @@ public void processChatMessage(Player player, Component message, String channel,
         if (hasGoodGroup) userPrimaryGroup = userPrimaryGroup.substring(0, 1).toUpperCase() + userPrimaryGroup.substring(1);
 
         boolean reserializer = DiscordSRV.config().getBoolean("Experiment_MCDiscordReserializer_ToDiscord");
+        boolean webhookMessageDelivery = config().getBoolean("Experiment_WebhookChatMessageDelivery");
 
-        String username = player.getName();
-        if (!reserializer) username = DiscordUtil.escapeMarkdown(username);
-
-        String discordMessage = (hasGoodGroup
-                ? LangUtil.Message.CHAT_TO_DISCORD.toString()
-                : LangUtil.Message.CHAT_TO_DISCORD_NO_PRIMARY_GROUP.toString())
-                .replaceAll("%time%|%date%", TimeUtil.timeStamp())
-                .replace("%channelname%", channel != null ? channel.substring(0, 1).toUpperCase() + channel.substring(1) : "")
-                .replace("%primarygroup%", userPrimaryGroup)
-                .replace("%username%", username)
-                .replace("%usernamenoescapes%", MessageUtil.strip(player.getName()))
-                .replace("%world%", player.getWorld().getName())
-                .replace("%worldalias%", MessageUtil.strip(MultiverseCoreHook.getWorldAlias(player.getWorld().getName())));
-        discordMessage = PlaceholderUtil.replacePlaceholdersToDiscord(discordMessage, player);
-
-        String displayName = MessageUtil.strip(player.getDisplayName());
-        displayName = DiscordUtil.escapeMarkdown(displayName);
         String discordMessageContent;
         if (reserializer) {
             discordMessageContent = MessageUtil.reserializeToDiscord(message);
         } else {
             discordMessageContent = MessageUtil.strip(MessageUtil.toLegacy(message));
         }
 
-        discordMessage = discordMessage
-                .replace("%displayname%", displayName)
-                .replace("%displaynamenoescapes%", MessageUtil.strip(player.getDisplayName()))
-                .replace("%message%", discordMessageContent);
-
-        discordMessage = processRegex(discordMessage);
-        if (discordMessage == null) return;
-
-        if (!reserializer) discordMessage = MessageUtil.strip(discordMessage);
+        //Modify the message's content with the declared Regexes
+        discordMessageContent = processRegex(discordMessageContent);
```

Comment:
Regexes are now only applied to the content rather than the entire message, they should still be applied to the entire content (when webhooks are not being used)

Reply:
d312f97a should fulfill this request